examples/low-level-api/resolve_did.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! A basic example that generates a DID Document, publishes it to the Tangle,
+//! and retrieves information through DID Document resolution/dereferencing.
+//!
+//! See also https://www.w3.org/TR/did-core/#did-resolution and https://www.w3.org/TR/did-core/#did-url-dereferencing
+//!
+//! cargo run --example resolution
+
+use identity::iota::Receipt;
+use identity::iota::ResolvedIotaDocument;
+use identity::iota::Resolver;
+use identity::iota_core::IotaDID;
+use identity::prelude::*;
+
+mod create_did;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+  // Create a signed DID Document and KeyPair (see create_did.rs).
+  let (document, _, _): (IotaDocument, KeyPair, Receipt) = create_did::run().await?;
```

Comment:
```suggestion
  let account: Account = Account::builder()
    .create_identity(IdentitySetup::default())
    .await?
```
In the new examples, we try to make them not depend on each others, and using the account makes it easy to do that, but maybe this requires moving the example to the account folder. So maybe this suggestion is optional for this PR but needed for the new example structure.

Reply:
Thank you @abdulmth for this insight. It makes a lot of sense to use the Account where possible, but I would indeed like to wait with making your suggested change until we have the new example structure in place. Will definitely make sure to remember this for later though :+1: